code/datums/gamemodes/crash.dm

```diff
@@ -111,7 +146,7 @@
 
 	announce_bioscans(TRUE, 0, FALSE, TRUE) // Announce exact information to the xenos.
 
-	addtimer(CALLBACK(src, .proc/add_larva), 5 MINUTES, TIMER_LOOP)
+	addtimer(CALLBACK(src, .proc/add_larva), 10 MINUTES, TIMER_LOOP)
```

Comment:
Increased the time a bit. If there are many more marines than xenos, xenos can get more larvas than just one. And if there are too many xenos, they don't get extra larvas.